CODE/style.css

```diff
@@ -0,0 +1,130 @@
+*,
+*:before,
+*:after {
+    box-sizing: border-box;
+}
+
+body {
+    background-color: #d2d3d3;
+
+}
+
+/* pull QUote*/
+blockquote {
+    text-align: left;
+    margin: 0;
+}
+
+
+header {
+    background-image: url(images/maxresdefault.jpg);
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+
+/*Title and Author*/
+h1 {
+    font-family: 'Oswald', sans-serif;
+    font-size: 60px;
+    text-align: center;
+    color: #ce5454f8;
+}
+
+h2 {
+    font-family: 'Oswald', sans-serif;
+    font-size: 45px;
+    text-align: center;
+    color: #ce5454f8;
+}
+
+h3 {
+    font-family: 'Oswald', sans-serif;
+    font-size: 30px;
+    text-align: left;
+}
+
+/*article text*/
+p1 {
+    columns: 100 px 1;
+    margin: -438px 20px 20px 270px;
+    float: left;
+    font-family: 'Libre Franklin', sans-serif;
+    font-size: 25px;
+    text-align: left;
+    line-height: 1.85;
+    padding: 30 20 50px;
+
+
+}
+
+p2 {
+    columns: 100 px 1;
+    margin: -195px 20px 0 270px;
+    float: left;
+    font-family: 'Libre Franklin', sans-serif;
+    font-size: 25px;
+    text-align: left;
+    line-height: 1.85;
+    padding: 30 20 50px;
+
+
+}
+
+p3 {
+    columns: 100 px 1;
+    margin: 0px 20px 0 30px;
+    float: left;
+    font-family: 'Libre Franklin', sans-serif;
+    font-size: 25px;
+    padding-bottom: 50px;
+    text-align: left;
+    line-height: 1.85;
+    padding: 30 20 50px;
+
+
+
+}
+
+p4 {
+    columns: 100 px 1;
+    margin: -5px 20px 0 30px;
+    float: left;
+    font-family: 'Libre Franklin', sans-serif;
+    font-size: 25px;
+    text-align: left;
+    line-height: 1.85;
+    padding: 30 20 50px;
+
+}
+
+/* caption at the end of the article*/
+figcaption {
+    font-family: 'Libre Franklin', sans-serif;
+    font-size: 10px;
+    color: #8b8895;
+}
+
+/**/
+.pquote {
+    font-family: 'Oswald', sans-serif;
+    font-size: 30px;
+    text-align: left;
+    float: left;
+    width: 100px;
+    font-weight: bold;
+    padding: 0;
+    margin: -50px 20px -10px 20px;
+}
+
+
+/**/
+.pquote {
```

Comment:
This is a repetition of the class you have above and, as a result, you have declared these properties multiple times -- e.g., `font-size` is declared 3 times on the `pquote` class: Lines 110, 123, 127.